pootle/core/templatetags/core.py

```diff
@@ -25,6 +25,13 @@ def to_js(value):
     return mark_safe('JSON.parse("%s")' % escapejs(jsonify(value)))
 
 
+@register.filter
+def map_to_lengths(value):
+    """Maps a list value by replacing each element with its length.
```

Comment:
this could do with a quick test

Reply:
Added a few basic tests. I hope I put them into the right directory.